exampleCourse/questions/examplesPanels/question.html

```diff
@@ -0,0 +1,77 @@
+<pl-question-panel>
+  PrairieLearn questions have three types of <i>panels</i> that are shown to the student:
+  <ol>
+    <li><b>Question panel:</b> shows the question prompt. The panel containing this text is the question panel. There is only one question panel and it is always visible.</li>
+    <li><b>Submission panel:</b> shows the student's submitted answer. There is one submission panel for each submitted answer, so the student can see the history of their attempts.</li>
+    <li><b>Answer panel:</b> shows the correct answer or other feedback to the student once they can no longer attempt the question. There is at most one answer panel.</li>
+  </ol>
+
+  <p>By default, the contents of <tt>question.html</tt> is shown in all three panels. Elements used in <tt>question.html</tt> are aware of which panel they are in. For example, the <tt>&lt;pl-multiple-choice&gt;</tt> element will let the student choose an option in the question panel, show the student the option they chose in the submission panel, and display the correct option in the answer panel.</p>
+
+  <h5>Elements to control panel display</h5>
+
+  <p>There are four elements that can be used to control whether text is shown in each panel:</p>
+  <ul>
+      <li><b><tt>&lt;pl-question-panel&gt;</tt></b> only displays its contents in the question panel. This should be used for the question prompt.</li>
+      <li><b><tt>&lt;pl-submission-panel&gt;</tt></b> only displays its contents in the submission panel. This could be used to give specific feedback on a particular student submission.</li>
+      <li><b><tt>&lt;pl-answer-panel&gt;</tt></b> only displays its contents in the answer panel. This could be used to give information about the correct answer.</li>
+      <li><b><tt>&lt;pl-hide-in-panel&gt;</tt></b> has attributes that allow content to be hidden in one or more panels.</li>
+  </ul>
+
+  <p>In normal usage, we put the question prompt in a <tt>&lt;pl-question-panel&gt;</tt> element and all inputs requesting student input outside of this element.</p>
+
+  <h5>Example HTML</h5>
+
+  <p>As an example, this question contains the follow panel elements.  Click the "Save &amp; Grade" button to make the submission and answer panels appear.</p>
+<pre>
+&lt;pl-question-panel&gt;
```

Comment:
I think the use of `&lt;`/`&gt;` is to avoid the tag being picked up. If that's not the case, it might be a bit better to wrap this within the new `<markdown></markdown>` tag.

Reply:
You are correct, this is just HTML escaping. Markdown would be an alternative, but would still require escaping to avoid switching into HTML.